pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -64,6 +64,8 @@ type FakePostFilterPlugin struct {
 	numViolatingVictim int
 }
 
+type FakePreemptionScorePostFilterPlugin struct{}
```

Comment:
move it above L91 for better readability. And also explain briefly this plugin's intent.